aya/src/programs/tc.rs

```diff
@@ -89,6 +97,12 @@ pub enum TcError {
     /// the clsact qdisc is already attached
     #[error("the clsact qdisc is already attached")]
     AlreadyAttached,
+    /// tcx links can only be attached to ingress or egress
+    #[error("tcx links can only be attached to ingress or egress, custom attachment: {0} is not supported")]
```

Comment:
@tamird Typically, tc programs are attached to the clsact qdisc in either the ingress or egress direction.  I've not tried this, but in theory, tc programs can also be attached to an arbitrary qdisc.  Tcx programs only support an engress or egress direction and don't attach to a qdisc.  So, I guess this error is trying to give the user some feedback about what custom attachment id was given.